Source/diablo.cpp

```diff
@@ -58,7 +58,7 @@ int frameend;
 int framerate;
 int framestart;
 #endif
-int FriendlyMode = 1; // weak
+BOOL FriendlyMode = FALSE;

```

Comment:
Looks like this was TRUE before.

Reply:
Maybe he likes auto-hostility in multiplayer ;)

Reply:
Oh my. Fixed.